src/tribler/core/upgrade/upgrade.py

```diff
@@ -107,6 +108,9 @@ def run(self):
         self.upgrade_pony_db_14to15()
         self.upgrade_knowledge_to_tribler_db()
 
+        migration = TriblerDatabaseMigrationChain(self.state_dir)
+        migration.execute()
```

Comment:
IMO, it is better to rename the variable to `migrations` or `migration_chain` here. Otherwise, we have confusing terminology and use the same terms with different meanings in different parts of the code.

It is better to have the following definitions:
* a migration chain consists of individual migrations
* an individual migration can possibly consist of several steps (usually only one)

Reply:
Agree, renamed to the `migration_chain`